.ci/test.sh

```diff
@@ -57,9 +57,13 @@ if [[ $TASK == "lint" ]]; then
             "r-lintr>=2.0"
     pip install --user cpplint isort mypy
     echo "Linting Python code"
+    echo "..pycodestyle"
     pycodestyle --ignore=E501,W503 --exclude=./.nuget,./external_libs . || exit -1
-    pydocstyle --convention=numpy --add-ignore=D105 --match-dir="^(?!^external_libs|test|example).*" --match="(?!^test_|setup).*\.py" . || exit -1
+    echo "..pydocstyle"
+    pydocstyle --convention=numpy --add-ignore=D105 --match-dir='^(?!^external_libs|test|example).*' --match='(?!^test_|setup).*\.py' . || exit -1
+    echo "..isort"
     isort . --check-only || exit -1
+    echo "..mypy"
```

Comment:
Can you please remove these changes adding new `echo` statements and moving the order of CI checks around? They don't seem related to this PR.

Reply:
fixed via 941dfce43418523067a3edf5bb93a1883fee38a2